snappass/templates/404snap.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="container">
+  <section>
+    <div class="page-header"><h1>Secret Not Found</h1></div>
+    <p>The secret you requested is not available. This typically indicates that one of the three below conditions has occurred:</p> 
+    <ol>
+         <li>The secret has already been retrieved.</li>
+         <li>The link was corrupted</li>
```

Comment:
To match the other items in the list:

```suggestion
         <li>The link was corrupted.</li>
```